packages/web-components/src/components/filter-panel/filter-panel-modal.ts

```diff
@@ -26,21 +27,81 @@ import { carbonElement as customElement } from '../../internal/vendor/@carbon/we
 const { prefix } = settings;
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+const WITHIN = Node.DOCUMENT_POSITION_CONTAINED_BY;
```

Comment:
This all looks like it's copy pasted over from somewhere else - that's okay, but I imagine there are ways we can bring this code in without duplicating. Lets schedule a time to pair up and see if we can figure out how to keep this DRY.